idd/Energy+.idd.in

```diff
@@ -12497,12 +12499,12 @@ InternalMass,
        \note To be matched with a construction in this input file
        \type object-list
        \object-list ConstructionNames
-  A3 , \field Zone Name
+  A3 , \field Zone or ZoneList Name
        \required-field
+       \type object-list
+       \object-list ZoneAndZoneListNames
        \note Zone the surface is a part of
        \note used to be Interior Environment
```

Comment:
If there are any other changes here - this note could be deleted. "Interior Environment" is from long long ago.